### DISCUSSION TITLE
UserPreferences not being saved for single grid?

### DISCUSSION THREAD

**Discussion by rchawla-alphabyte** [serenity-is/Serenity#5828] [Help]
My user preferences are being saved for all my other grids, however it's not being saved in the table UserPreferences for one _specific_ grid.

I have added the method manually in the grid file however, no luck.

![image](https://user-images.githubusercontent.com/76563021/123438548-c9ac6680-d59e-11eb-83ad-25d98b5c9739.png)

I have verified that the endpoint is being called as well, any suggestions would be appreciated!

Thanks

**Comment by VictorTomaili** [comment]
https://serenity.is/docs/grids/persisting_settings
